index.js

```diff
@@ -201,5 +244,10 @@ function isShareCodePattern(code) {
 module.exports = {
 	encrypt: encrypt,
 	decrypt: decrypt,
-	isShareCodePattern: isShareCodePattern
+	isShareCodePattern: isShareCodePattern,
+	_removeHyphens: removeHyphens,
```

Comment:
prefixing with _ was the quickest non 100% crap way I could find of exposing the inner functions to allow for better testing whilst hoping that no-one ever uses those functions in the wild.